homeassistant/components/apple_tv/media_player.py

```diff
@@ -69,6 +70,7 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     """Load Apple TV media player based on a config entry."""
     name = config_entry.data[CONF_NAME]
     manager = hass.data[DOMAIN][config_entry.unique_id]
+    await manager.connect()
```

Comment:
Keep the device always connected since - https://github.com/home-assistant/core/pull/51520/files#r646072824

Reply:
Link doesn't work anymore and I don't remember... why is this? Seems backwards that a platform would trigger this, it should be handled by the component once all platforms have been loaded properly. Which I think is the case now? I guess the problem is the "start off" option?